bugfix: whois timeout (#21315)
demisto/content: Packs/Whois/ReleaseNotes/1_4_0.md

#### Integrations
##### Whois

> - Fixed an issue that caused domain command to timeout for curtain domains

Reviewer: ```suggestion
- Fixed an issue where **domain** command timed out for certain domains.
```
